match/lib/match.rb

```diff
@@ -29,4 +29,21 @@ def self.environments
   def self.enterprise?
     ENV["MATCH_FORCE_ENTERPRISE"]
   end
+
+  def self.type_is_enterprise?(type)
```

Comment:
Could you add a comment above this method and the `self.enterprise?` to describe what the method returns?